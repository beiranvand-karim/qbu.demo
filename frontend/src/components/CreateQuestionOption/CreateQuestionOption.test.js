/* global describe, it, jest, expect*/
import {shallow, mount} from "enzyme"
import {CreateQuestionOption} from "./"
import React from "react"
import renderer from "react-test-renderer"
import {MemoryRouter} from "react-router-dom"


describe('<CreateQuestionOption />', () => {

   it('should render without crashing', () => {
      shallow(<CreateQuestionOption/>)
   });

   // todo write unit test fot testing image upload

   it('should matches the snapshot', function () {
      const tree = renderer.create(<MemoryRouter><CreateQuestionOption/></MemoryRouter>).toJSON();
      expect(tree).toMatchSnapshot()
   });
});