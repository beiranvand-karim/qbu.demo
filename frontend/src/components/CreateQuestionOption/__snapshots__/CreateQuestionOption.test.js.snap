// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<CreateQuestionOption /> should matches the snapshot 1`] = `
<div>
  <input
    onChange={[Function]}
    type="file"
  />
</div>
`;
